src/main/java/org/opentripplanner/transit/raptor/rangeraptor/transit/TransitCalculator.java

```diff
@@ -204,4 +208,6 @@
      * Same as {@link #boardingPossibleAt(RaptorTripPattern, int)}, but for switched alighting/boarding.
      */
     boolean alightingPossibleAt(RaptorTripPattern pattern, int stopPos);
+
+    Iterator<? extends RaptorTransfer> getTransfers(int fromStop);
```

Comment:
Should we pass in the transit data as a parameter, rather than embed it into the calculator? That is how most other data is used in conjunction with the calculator.

Reply:
Yes, it would be cleaner -- I've update the code.